packages/components/snackbar/src/Snackbar.tsx

```diff
@@ -55,7 +55,13 @@ export const Snackbar = forwardRef<HTMLDivElement, SnackbarProps>(
     useEffect(() => {
       addProvider(ref)
 
-      return () => deleteProvider(ref)
+      return () => {
+        for (const toast of getGlobalSnackBarQueue().visibleToasts) {
+          toast.animation = undefined
```

Comment:
More sugar than real improvement... This allows us to ensure `toast.animation` state is properly resetted on unmount.